config/standard_ingest.yml.sample

```diff
@@ -4,6 +4,7 @@
         - Thumbs.db
         - metadata.txt
     :targets: dpc_targets
+    :intermediate_files: intermediate_files
```

Comment:
@dchandekstark Test suite ran successfully on my workstation without the additional line in my config/standard_ingest.yml so I'm thinking it's not required for testing.